File: Pattern/pattern.py
import math
 
def printPattern(r):
     
    for i in range((2 * r)+1):
        # change below line to pass all test cases
        for j in range(r + 1):  
            # change below line to pass all test cases
            dist = math.sqrt((i - r) * (i - r) + (j + r) * (j + r))
            if (dist > r - 0.5 and dist < r + 0.5):
                print("*",end="")
            else:
                print(" ",end="")     
        print()

r = int(input())
# change below line to pass all test cases
printPattern()

File: Pattern/pattern.c
void printPattern(int r) {
 
  float dist;
  for (int i = 0; i <= 2 * r; i++) {
    // change below line to pass all test cases
    for (int j = 0; j <= r; j++) {
      // change below line to pass all test cases
      dist = sqrt((i - r) * (i - r) + (j + r) * (j + r));
      if (dist > r - 0.5 && dist < r + 0.5)
        printf("*");
      else
        printf(" ");
    }
    printf("\n");
  }
}
 
int main() {
  int r;
  scanf("%d", &r);
  // change below line to pass all test cases
  printPattern();
  return 0;
}